setup.cfg

```diff
@@ -1,2 +1,35 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = Flask-DebugToolbar
+version = 0.12.dev0
+author = Michael van Tellingen
+author_email = michaelvantellingen@gmail.com
+maintainer = Matt Good
+maintainer_email = matt@matt-good.net
+description = A toolbar overlay for debugging Flask applications.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+keywords = flask, debug, toolbar
+url = https://github.com/flask-debugtoolbar/flask-debugtoolbar
+project_urls =
+    Changelog = https://github.com/jeffwidman/cqlsh#changelog
+    Documentation = https://github.com/flask-debugtoolbar/flask-debugtoolbar/blob/master/CHANGES.rst
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Web Environment
+    Framework :: Flask
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+packages = find:
+package_dir = = flask_debugtoolbar
```

Comment:
I realize this looks confusing, but I copied it straight from Flask where it's been present for 2 years: https://github.com/pallets/flask/blob/41aaaf7fa07f28ef15fd1d343142e96440889a8b/setup.cfg#L36